### ISSUE TITLE
QR code broken - getURL() function fix within

### ISSUE THREAD

**Issue by dustinbolton** [chregu/GoogleAuthenticator.php#3]
It looks like both the QR code URL and some formatting changes are needed to get this to work correctly.  Working function as of November 29, 2012:

<pre>
public function getUrl($user, $hostname, $secret) {
        $url =  sprintf("otpauth://totp/%s@%s?secret=%s", $user, $hostname, $secret);
        $encoder = "https://chart.googleapis.com/chart?chs=200x200&chld=M|0&cht=qr&chl=";
        $encoderURL = $encoder . urlencode( sprintf( "otpauth://totp/%s@%s?secret=%s", $user, $hostname, $secret) );
        
        return $encoderURL;
        
    }
</pre>

**Comment by andreszs** [comment]
You are right, the code should be updated to reflect the new Google APIs URL.
